src/garage/tf/baselines/gaussian_mlp_baseline_with_model.py

```diff
@@ -1,71 +0,0 @@
-"""A value function (baseline) based on a GaussianMLP model."""
```

Comment:
can you use `git rm` and `git mv` to mark these files as deleted/moved rather than edited/deleted? And similarly for other replaced files in this PR

e.g.
```sh
git rm src/garage/tf/baselines/gaussian_mlp_baseline.py
git rm tests/garage/tf/baselines/test_gaussian_mlp_baseline.py
git mv src/garage/tf/baselines/gaussian_mlp_baseline_with_model.py src/garage/tf/baselines/gaussian_mlp_baseline.py
```